Headers/AppLovinSDK.framework/Headers/ALSdkSettings.h

```diff
@@ -0,0 +1,112 @@
+//
+//  ALSdkSettings.h
+//  AppLovinSDK
+//
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This class contains settings that enable the AppLovin consent flow.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flowMAX Integration Guide ⇒ iOS ⇒ Consent Flow</a>
+ */
+@interface ALConsentFlowSettings : NSObject
+
+/**
+ * Whether or not the flow should be enabled. You must also provide your privacy policy and terms of service URLs in this object, and you must provide a
+ * `NSUserTrackingUsageDescription` string in your `Info.plist` file.
+ *
+ * This defaults to the value that you entered into your `Info.plist` file via `AppLovinConsentFlowInfo`⇒`AppLovinConsentFlowEnabled`.
+ *
+ * @see <a href="https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#info.plist">MAX Integration Guide ⇒ iOS ⇒ Consent Flow ⇒ Info.plist</a>
+ */
+ // [PLP] "should be enabled" by whom?
```

Comment:
By the publisher. Related doc [here](https://dash.applovin.com/documentation/mediation/ios/getting-started/consent-flow#enabling-max-consent-flow).